evap/staff/tests/test_importers.py

```diff
@@ -775,8 +819,38 @@ def test_existing_participation(self):
         )
         self.assertFalse(importer_log.has_errors())
 
+    @override_settings(IMPORTER_COURSE_NAME_SIMILARITY_WARNING_THRESHOLD=0.8)
+    def test_course_name_with_typo(self):
+        # Add a typo in one english course name as well
```

Comment:
Should we test that this doesn't occur without the additional typo? Or is that covered by the other tests?

Reply:
Hmm, good question, idk. This test is the only one A) with the typo in one line and B) with the lowered threshold, so yeah, it probably makes sense to assert that the import passes without the typo.

Reply:
However, this test asserts that the warning is triggered for exactly three out of the 35 lines, so at the same time it asserts that the other 32 lines pass without a warning, so I think the current test logic is fine. What do you think?

Reply:
Fair enough, even within the six names that do collide, there aren't any collisions outside of the pairs, so that should suffice